Libraries/Opc.Ua.Client/SessionReconnectHandler.cs

```diff
@@ -83,32 +120,103 @@ protected virtual void Dispose(bool disposing)
         /// <value>The session.</value>
         public ISession Session => m_session;
 
+        /// <summary>
+        /// The internal state of the reconnect handler.
+        /// </summary>
+        public ReconnectState State
+        {
+            get
+            {
+                lock (m_lock)
+                {
+                    if (m_reconnectTimer == null)
```

Comment:
It should be safe to always return m_state, the special handling of m_reconnectTimer == null seems unnecessary